Helper/Data.php

```diff
@@ -1885,6 +1885,25 @@ public function buildThreeDS2ProcessResponseJson($type = null, $token = null)
 
         return json_encode($response);
     }
+    /**
+     * @param string|null $type
+     * @param string|null $token
+     * @return string
+     */
+    public function buildThreeDSProcessResponseJson($type = null, $token = null)
```

Comment:
Why don't you use the buildThreeDS2ProcessResponseJson() instead of adding this function?

Reply:
The buildThreeDS2ProcessResponseJson  adds to the response 
$response['threeDS2'] = true;